README.md

```diff
@@ -74,6 +74,7 @@ We have a roadmap for future development work `align-anything`:
 
 # News
 
+- 2024-09-12 🔥 We have supported SFT, DPO and PPO for text-video-image to text models!

```

Comment:
Add the corresponding section to [README_zh-CN.md](https://github.com/PKU-Alignment/align-anything/blob/main/README_zh-CN.md)